src/store/assets/tokens.ts

```diff
@@ -73,11 +73,10 @@ export const nonFungibleTokensState = atom(get => {
 });
 export const stxTokenState = atom(get => {
   const balances = get(accountBalancesState);
-  if (!balances || balances.stx.balance.isEqualTo(0)) return;
   return {
     type: 'stx',
     contractAddress: '',
-    balance: new BigNumber(balances.stx.balance),
+    balance: balances?.stx.balance ? new BigNumber(balances?.stx.balance) : 0,
```

Comment:
This should probably be the `currentAccountAvailableStxBalanceState`, right?

Reply:
Yep, that is probably what we want here. I will update it.